app/src/main/java/org/oppia/app/home/HomeFragmentPresenter.kt

```diff
@@ -97,11 +95,6 @@ class HomeFragmentPresenter @Inject constructor(
     })
   }
```

Comment:
playExplorationButton is unused. Please Let me know what to do with this.

Reply:
Keep it for now, otherwise it will start giving error in test cases that are using that button, which make it hard to run test cases in other files too.

Reply:
Ok